docs/sources/docker-hub/repos.md

```diff
@@ -48,24 +43,9 @@ private to public.
 > A collaborator cannot add other collaborators. Only the owner of
 > the repository has administrative access.
 
-You can also collaborate on Docker Hub with organizations and groups.
-You can read more about that [here](accounts/).
-
-## Official Repositories
-
-The Docker Hub contains a number of [Official
-Repositories](http://registry.hub.docker.com/official). These are
-certified repositories from vendors and contributors to Docker. They
-contain Docker images from vendors like Canonical, Oracle, and Red Hat
-that you can use to build applications and services.
-
-If you use Official Repositories you know you're using a supported,
-optimized and up-to-date image to power your applications.
-
-> **Note:**
-> If you would like to contribute an Official Repository for your
-> organization, see [Official Repositories on Docker
-> Hub](/docker-hub/official_repos) for more information.
+You can also assign more granular collaborator rights ("Read", "Write", or "Admin")
+on Docker Hub by using organizations and groups. You can read more about that
+[here](accounts/).
```

Comment:
Avoid using "here" for link text. Use descriptive text instead. E.g., For more information, see the [accounts documentation](http://etc).